cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/provisioner/dataproc/DataprocClient.java

```diff
@@ -701,7 +730,7 @@ private boolean isInternalIPOnly(boolean privateInstance, boolean sshRuntimeMoni
    * @return a {@link Collection} of tags that need to be added to the VM to have those firewall rules applies
    * @throws IOException If failed to discover those firewall rules
    */
-  private List<String> getFirewallTargetTags(boolean useInternalIP) throws IOException {
+  private List<String> getFirewallTargetTags(Network network, boolean useInternalIP) throws IOException {
```

Comment:
Similarly here no need to pass `network` explicitly to the method.

Reply:
Same reason as above.